<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>505</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>807</width>
    <height>505</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: beige;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>460</y>
      <width>57</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: white;
    color: navy;
    border: none;
    border-radius: 4px;
    padding: 6px;
}</string>
    </property>
    <property name="text">
     <string>首页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>460</y>
      <width>70</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: white;
    color: navy;
    border: none;
    border-radius: 4px;
    padding: 6px;
}</string>
    </property>
    <property name="text">
     <string>日程表</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>460</y>
      <width>57</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: white;
    color: navy;
    border: none;
    border-radius: 4px;
    padding: 6px;
}</string>
    </property>
    <property name="text">
     <string>我的</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>460</y>
      <width>83</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: white;
    color: navy;
    border: none;
    border-radius: 4px;
    padding: 6px;
}</string>
    </property>
    <property name="text">
     <string>数据统计</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>783</width>
      <height>401</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QToolButton" name="toolButton">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>160</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton {
    background-color: white;
    color: navy;
    font-size: 12px;
    border: 10px lightsteelblue;
    border-radius: 10px;
    padding: 6px;
}</string>
      </property>
      <property name="text">
       <string>选择计划</string>
      </property>
      <property name="popupMode">
       <enum>QToolButton::ToolButtonPopupMode::InstantPopup</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>310</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton {
    background-color: white;
    color: navy;
    font-size: 12px;
    border: powderblue;
    border-radius: 10px;
    padding: 6px;
}</string>
      </property>
      <property name="text">
       <string>未完成</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_3">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>160</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton {
    background-color: white;
    color: navy;
    font-size: 12px;
    border: 10px powderblue;
    border-radius: 10px;
    padding: 6px;
}</string>
      </property>
      <property name="text">
       <string>制定计划</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_4">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>310</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton {
    background-color: white;
    color: navy;
    font-size: 12px;
    border: powderblue;
    border-radius: 10px;
    padding: 6px;
}</string>
      </property>
      <property name="text">
       <string>历史记录</string>
      </property>
     </widget>
     <widget class="QLabel" name="choose_img">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>50</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="plan_img">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>50</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="notdone_img">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>200</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="record_img">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>200</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
    <widget class="QWidget" name="page_3">
     <widget class="QTextEdit" name="textEdit_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>140</y>
        <width>104</width>
        <height>74</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'.AppleSystemUIFont'; font-size:36pt;&quot;&gt;数据&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QTextEdit" name="textEdit_3">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>140</y>
        <width>141</width>
        <height>74</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'.AppleSystemUIFont'; font-size:36pt;&quot;&gt;profile&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>450</y>
      <width>811</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
	background-color: wheat;
	border: none;
}</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>10</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: navy;
}</string>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <zorder>graphicsView</zorder>
   <zorder>pushButton_1</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>stackedWidget</zorder>
   <zorder>quitButton</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
